powerquery-docs/connectors/kx-kdb-insights-enterprise.md

```diff
@@ -0,0 +1,97 @@
+---
+title: KX kdb Insights Enterprise Power Query Connector (Beta)
+description: Provides basic information, prerequisites, and instructions on how to connect to kdb Insights Enterprise
+author: liznorris
+ms.date: 11/15/2024
+ms.author: dougklo
+---
+
+# KX kdb Insights Enterprise Power Query Connector (Beta)
+
+> [!NOTE]
+> The following connector article is provided by KX Systems Inc., the owner of this connector and a member of the Microsoft Power Query Connector Certification Program. If you have questions regarding the content of this article or have changes you would like to see made to this article, visit the [KX](https://www.kx.com) website and use the support channels there.
+
+
+## Summary
+
+| Item	                        | Description                 |
+| ------------------------------ | --------------------------- |
+| Release State                  | General Availability        |
+| Products                       | Power BI (Semantic models)<br>Power BI (Dataflows)<br>Fabric (Dataflow Gen2)                   |
```

Comment:
The supported products on this line are Power BI (Semantic models), Power BI (Dataflows), and Fabric (Dataflow Gen2). However, in the index.md file above, Power Apps (Dataflows) and Customer Insights (Dataflows) are also listed as supported. Please fix this discrepancy.